sentry_streams/src/messages.rs

```diff
@@ -251,7 +253,7 @@ pub enum PyStreamingMessage {
 
 impl Into<PyStreamingMessage> for Py<PyAny> {
     fn into(self) -> PyStreamingMessage {
-        Python::with_gil(|py| {
+        traced_with_gil("into pystreaming", |py| {
```

Comment:
I would use the trace_with_gil function in unit tests as  well.

Reply:
done